### ISSUE TITLE
scale out

### ISSUE THREAD

**Issue by gedw99** [golift/subscribe#18]
you can probably use NATS Jetstream to scale out.

its a different API etc , but its stateful and can be clustered etc.

**Comment by davidnewhall** [comment]
Yeah this module is pretty simple. I only ever used it in one place and never came back to make it better. Probably other things out there that can easily replace it.
